error-503.html

```diff
@@ -7,125 +7,159 @@
     </style>
   </head>
 
   <body>
-    <div class="notification-banner">
-      <p class="notification-banner--inner">This is a new service. Go <a href="http://www.nhs.uk/" class="notification-banner--link">back to NHS Choices.</a></p>
-    </div>
+      <script>document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
+      <!-- <script>(function(d){d.className=d.className.replace(/\bno-js\b/,'js-enabled')})(document.documentElement);</script> -->
+      <div class="skiplinks">
+        <div class="skiplinks--inner">
+          <a href="#content" class="skiplinks--link">Skip to main content</a>
+        </div>
+      </div>
 
-    <header role="banner">
-      <div class="global-header">
-        <div class="global-header--inner">
-          <a href="/" class="global-header--link" title="Go to the NHS.UK homepage">
-            <img src="images/logotype-nhs-mono.png" alt="">
-          </a>
+      <div class="notification-banner notification-banner--js-hidden" id="global-cookies-banner">
+        <p class="notification-banner--inner">
+        NHS.UK uses cookies to make the site simpler.
+        <a href="https://beta.nhs.uk/help/cookies">Find out more about cookies</a>
+        </p>
+      </div>
+
+      <div class="notification-banner notification-banner--hiviz js-feedback-banner">
+    <p class="notification-banner--inner">
+      This is new. <a class="back-to-choices" href="https://www.nhs.uk/Service-Search/Pharmacy/LocationSearch/10?nobeta=true">Find a pharmacy using our old service.</a>
+    </p>
+  </div>
+
+
+
+
+      <header class="site-header">
+    <div class="site-header__top">
+      <div class="site-header__brand">
+        <!-- noindex -->
+        <a href="/">
+          <img
+            src="https://assets.nhs.uk/images/choices-logo.png" srcset="
+                https://assets.nhs.uk/images/choices-logo.png 1x,
+                https://assets.nhs.uk/images/choices-logo@2x.png 2x,
+                https://assets.nhs.uk/images/choices-logo@3x.png 3x,"
```

Comment:
The comma at the end of can be removed here as it is thought of as an empty element.

Reply:
removed